src/modules/Elsa.EntityFrameworkCore.MySql/Migrations/Alterations/20240312145108_V3_1.cs

```diff
@@ -10,6 +10,12 @@ public partial class V3_1 : Migration
         /// <inheritdoc />
         protected override void Up(MigrationBuilder migrationBuilder)
         {
+            migrationBuilder.RenameColumn(
```

Comment:
I don't think so - nobody is using alterations right now. We are only  just starting to test it out, so there's no data - or if there is, it would be insignificant.